scaleway/helpers.go

```diff
@@ -216,7 +216,7 @@ func withStoppedServer(scaleway *api.API, serverID string, run func(*api.Server)
 
 // parseLocalizedID parses a localizedID and extracts the resource locality and id.
 func parseLocalizedID(localizedID string) (locality string, ID string, err error) {
-	tab := strings.SplitN(localizedID, "/", 2)
```

Comment:
this should remain 2

Reply:
Are we sure ? 

This can lead to errors difficult to diagnose, in case we have something like `zone/zone/ID`.

Do we really want to support `/` in IDs ? Is there a real use case for this ?

Reply:
we want to support `zone/ID` or `ID` and only these syntax

Reply:
Do we really want to allow `/` in ID ? 

Using `tab := strings.SplitN(localizedID, "/", 2)`, `my_zone/my_zone/my_ID` will succeed with `locality=my_zone, ID=my_zone/my_ID`: we want it to fail.

Using `-1`, and with a string containing 2 `/`, the function will fail because `len(tab)==3`.